content/datadog.textile

```diff
@@ -0,0 +1,83 @@
+
+If you're on the Ably "Pro plan":/pricing/pro, you can enable a limited Datadog-lite integration. "Standard plan":/pricing/standard users may request a 30-day trial to evaluate Datadog-lite before upgrading. This version includes a smaller set of metrics but still provides visibility into your Ably usage. If you need detailed analytics, per-channel breakdowns, custom dashboards, or advanced logging, upgrade to the full Datadog integration available on the Enterprise plan.
+
+|_. Feature |_. Datadog-lite |_. Full Datadog (Enterprise) |
+| Messages | ✅ | ✅ |
+| Connections | ✅ | ✅ |
+| Channels | ✅ | ✅ |
+| Bandwidth | ✅ | ✅ |
+| Message size | ✅ | ✅ |
+| Message direction breakdown (Inbound vs. Outbound) | ❌ | ✅ |
+| Per-Channel metrics | ❌ | ✅ |
+| Usage analytics & trends | ❌ | ✅ |
+| Custom dashboards & alerts | ❌ | ✅ |
+| Advanced logging & debugging | ❌ | ✅ |
+
+
+Ably includes @AppID@ and @AccountID@ tags with each metric to help with data organization. These ID tags enable you to filter, group, and analyze information efficiently, making it more manageable to spot trends and diagnose issues in your Datadog "dashboard":https://docs.datadoghq.com/dashboards/.
+
+To connect Ably and Datadog, you must authorize the integration through Datadog's "OAuth":https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/ flow. This process requires the @api_keys_write@ scope, allowing Ably to push data to your Datadog account. Once the integration is active, Datadog provides a default Ably "dashboard":https://docs.datadoghq.com/integrations/ably/ so that you can monitor key metrics without extra setup.
+
+Every 60 seconds, Ably streams "statistics":https://ably.com/docs/metadata-stats/stats to the Datadog API using the generated API key. Metrics come from channels like @[meta]stats.minute@ and include App ID and Account ID tags. Datadog then displays these metrics in the Ably dashboard, giving you a clear view of messages, channels, and connections.
+
+h2. Set up the Ably Datadog integration
+
+Initiate the integration from the Datadog:
+
+# In Datadog, go to *Integrations*, find the *Ably* tile, and click *Install Integration*.
+# Click *Connect Accounts* to start the authorization process. Datadog redirects you to Ably.
+# Log in to your *Ably* account.
+# Select your application from the *Your Apps* page.
+# In the Ably dashboard, navigate to *Integrations* and select *Connect to Datadog*.
+# When prompted in Datadog, authorize Ably using *OAuth* to grant access. The required authorization scope is: @api_keys_write@.
+# After completing authorization, return to the Ably dashboard. A confirmation banner appears, along with a *See Dashboard* button that takes you to your Datadog dashboard.
+
+h3. Troubleshoot
+
+If the provided information does not resolve your issue, contact Ably "support":https://ably.com/support. When contacting, include details such as your app ID, and any relevant logs to help troubleshoot.
```

Comment:
This sort of message would normally go after the troubleshooting sections, but I'm not sure if it's relevant to include.